hermes/probe/create/create.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Alicja Kwiecinska, GitHub: alicjakwie
+//
+// create contains all of the logic necessary to create the files with randomized contents in GCS
+package create
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"math/rand"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+
+	pb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+const (
+	FileNamePrefixSize      = 9    // len("Hermes_ID")
+	begin                   = 1    // begin is the ID of the first HermesFile.
+	end                     = 50   // end is the ID of the last HermesFile.
+	maxFileSizeBytes        = 1000 // maxFileSizeBytes is the maximum allowed file size in bytes.
+	hermesAPILatencySeconds = "hermes_api_latency_s"
+)
+
+type randomFile struct {
+	id   int32 // id is a positive integer in the range [begin,end]
+	size int   // size is the file size in bytes.
+}
+
+type randomFileReader struct {
+	size int // size in bytes
+	i    int // currently reading this byte
+	rand *rand.Rand
+}
+
+// Read implements the standard Read interface, it reads data from a randomFileReader.
+// Argument:
+//      buf: a byte slice serves as a buffer
+// Returns:
+//	n:  the size of the buffer buf following a Read operation
+//	err: error it should be nil when not done reading and io.EOF once the whole file contents have been read
```

Comment:
nit: align the margins of your arguments with those of your returns.
```suggestion
// Argument:
//	buf: a byte slice serves as a buffer
// Returns:
//	n:  the size of the buffer buf following a Read operation
//	err: error it should be nil when not done reading and io.EOF once the whole file contents have been read
```